packages/blade/src/components/Typography/Code/__tests__/Code.web.test.tsx

```diff
@@ -4,17 +4,23 @@ import renderWithTheme from '~src/_helpers/testing/renderWithTheme.web';
 
 describe('<Code />', () => {
   it('should render Code with default properties', () => {
-    const { container } = renderWithTheme(<Code>TEST_TOKEN</Code>);
+    const { container, getByText } = renderWithTheme(<Code>TEST_TOKEN</Code>);
+    expect(getByText('TEST_TOKEN')).toHaveStyle('font-size: 0.75rem');
+    expect(container.querySelector('span')).toHaveStyle('padding: 0px 4px');
```

Comment:
These two assertions might be overkill

Reply:
Actually the padding is important in this case since if we break padding it also breaks the line heights and alignments.

Reply:
Oh just realised I added it to all tests. Kept it only in first test and removed it from other tests.